test_requirements.txt

```diff
@@ -1,3 +1,4 @@
 -r requirements.txt
 coverage==3.7.1
 tox==1.8.1
+mock==1.0.1
```

Comment:
My bad, I am still using Python 2.7.x to enjoy python libs and django apps that do not support Python 3.x yet.